src/components/user/sign-up/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {
+  Grid, Row, Col, Alert,
+} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import { signUpAPI } from '../../../store/actions/sign-up';
+import SignUpForm from './sign-up-form';
+
+class SignUp extends React.Component {
+  constructor(props) {
+    super(props);
```

Comment:
You only need to pass in `props` here if you're using it in the constructor: https://stackoverflow.com/questions/30571875/whats-the-difference-between-super-and-superprops-in-react-when-using-e

Reply:
Never knew this. Thanks.